src/web/forms/fields/AttachmentInput.js

```diff
@@ -31,46 +31,57 @@ const AttachmentInputComponent = ({
   multiple: ?boolean,
   addAttachments: (attachments: Object, meta: Array<Object>) => void,
 }) => (
-  <p className="control">
-    <input
-      type="file"
-      accept={accept}
-      multiple={multiple}
-      onChange={e => {
-        const files = e.target.files;
+  <div className="field">
+    <label className="file-label">
+      <div className="file is-small is-boxed">
+        <input
+          className="file-input"
+          type="file"
+          accept={accept}
+          multiple={multiple}
+          onChange={e => {
+            const files = e.target.files;
+            const meta = Array(files.length);
+            const attachments = {};
+            const idPromises = Array(files.length);
 
-        const meta = Array(files.length);
-        const attachments = {};
-        const idPromises = Array(files.length);
+            for (let i = 0; i < files.length; i++) {
```

Comment:
Why did you change it?
Yes, this is no problem and, in practice, these are equivalent and there is no strong reason to choose the one.
Therefore I think I can't find the reason to change it.